capi/bind_gen/src/c.rs

```diff
@@ -16,7 +16,7 @@ fn get_type(ty: &Type) -> Cow<'_, str> {
         "u32" => "uint32_t",
         "u64" => "uint64_t",
         "usize" => "size_t",
-        "isize" => "ssize_t",
+        "isize" => "ptrdiff_t",
```

Comment:
You can do the same change in `capi/bind_gen/src/swift/header.rs`.

Reply:
Yeah please apply it there too.